pyproject.toml

```diff
@@ -62,6 +62,7 @@ exclude = '''
     | build
     | dist
   )/
+  | _version.py
```

Comment:
This file is auto-generated by `setuptools-scm` and should not be linted

Reply:
Full disclosure: I don't understand this file.  But if it's auto-generated and @bjlittle is happy with it then it should be 👍